Dockerfile

```diff
@@ -1,13 +1,13 @@
 # Keep hub.Dockerfile aligned to this file as far as possible
 ARG JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom"
-ARG APP_INSIGHTS_AGENT_VERSION=2.3.1-SNAPSHOT
+ARG APP_INSIGHTS_AGENT_VERSION=2.4.1
```

Comment:
bump to latest 2.5.1-BETA (https://github.com/hmcts/ccd-definition-store-api/pull/458)

Reply:
mmh, done